tox.ini

```diff
@@ -15,12 +15,13 @@ deps =
    coverage
    isort
    mccabe
+   pytest
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m unittest discover -s {envsitepackagesdir}/pylint/test/ -p {posargs:*test_*}.py
+    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/
```

Comment:
I think @rogalski has a fair point. Don't remove ``{posargs}`` here. We can use it to pass down flags and values to pytest, as we did with unittest initially, through tox.